docs/job/push-to-docker.md

```diff
@@ -22,6 +25,9 @@ workflows:
           image: "quay.io/giantswarm/REPOSITORY"
           username_envar: "QUAY_USERNAME"
           password_envar: "QUAY_PASSWORD"
+          build-context: "."
```

Comment:
Maybe `build-path`?

```
$ docker build --help | head -n 2

Usage:  docker build [OPTIONS] PATH | URL | -
```

Reply:
Docker calls this `context` so if you don't have a strong preference to call this `build-path` I think `build-context` is better.